templates/zerver/server_needs_upgrade.html

```diff
@@ -0,0 +1,41 @@
+{% extends "zerver/portico.html" %}
+
+{% block content %}
+
+<div class="error_page">
+    <div class="container">
+        <div class="row-fluid">
+            <img src="/static/images/400art.svg" alt=""/>
+            <div class="errorbox config-error">
+                <div class="errorcontent">
+                    <h1 class="lead">{{ _('Server upgrade required') }}</h1>
+                    <p>
+                        {% trans %}
+                        You are running a Zulip server that needs upgrade.
+                        {% endtrans %}
+                    </p>
+
+                    <p>
+                        {% trans %}
+                        Zulip servers that had not been updated for the past one year
+                        or more would need an upgrade.
+                        {% endtrans %}
+                    </p>
+
+                    <p>
+                        <a
+                          href="https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html#upgrading-to-a-release"
+                          target="_blank"
+                          rel="noopener noreferrer"
+                          >
+                            {{ _("Upgrade to a release.") }}
+                        </a>
+                    </p>
+                </div>
+
+            </div>
+        </div>
+    </div>
+</div>
+
+{% endblock %}
```

Comment:
Why do we need a file here?  I thought we were going to use the `panels.js` system to warn, which would not require a new backend template file.